### PR TITLE
Aswathy/fix: updated the business terms conditions pdf for Spanish row users

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   Update the pdf of business partners terms and conditions for spanish ROW users

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/terms-and-conditions/_business-grid.tsx [hunk 1]
```diff
@@ -78,7 +85,9 @@ const BusinessGrid = () => {
                     title="_t_General terms of use_t_"
                     content="_t_Terms and ethical standards for all our affiliates, introducing brokers, API users, and payment agents_t_"
                     url={
-                        is_row
+                        is_row && language === 'es'
```
Comment (konstantinosG-deriv) [inline src/pages/terms-and-conditions/_business-grid.tsx:88]: What if a user is from EU and selects Spanish? Then they will get this file _/tnc/business-partners-general-terms-eu.pdf_ but shouldn't they get /tnc/business-partners-general-terms-**es**.pdf?

File: src/pages/terms-and-conditions/_business-grid.tsx [hunk 2]
```diff
@@ -62,7 +62,14 @@ const PartnersGuidePdf = () => {
 }
 
 const BusinessGrid = () => {
+    const general_terms_url = `/tnc/business-partners-general-terms-es.pdf`
```
Comment (konstantinosG-deriv) [inline src/pages/terms-and-conditions/_business-grid.tsx:65]: Could put **_es** or **_spanish** at the end of these variable names for clarity

File: src/pages/terms-and-conditions/_business-grid.tsx (hunk 1 above)
Comment (aswathy-deriv) [reply src/pages/terms-and-conditions/_business-grid.tsx:88]: No as of now they have not yet done the EU for spanish..so we will show the EU pdf..once that's done they will update us

File: src/pages/terms-and-conditions/_business-grid.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/pages/terms-and-conditions/_business-grid.tsx:65]: Other Pdfs which is translated is named like this..so as a common gave the name the same

Comment (konstantinosG-deriv) [reply src/pages/terms-and-conditions/_business-grid.tsx:65]: Pdf file is okay, I meant this **general_terms_url** maybe to **general_terms_url_es**

**Review by hasan-deriv — APPROVED** [review]

File: src/pages/terms-and-conditions/_business-grid.tsx (hunk 2 above)
Comment (aswathy-deriv) [reply src/pages/terms-and-conditions/_business-grid.tsx:65]: In future,other languages will also come up..so we should use a constant variable..should not name it based on the language..only the url link will change..but the variable name should remain constant for all the languages

**Review by mitra-deriv — APPROVED** [review]

**Review by konstantinosG-deriv — APPROVED** [review]

**Pull request merged**